### PR TITLE
Adding favicon to match logo.

### PR DESCRIPTION
**Pull request by dhermes**
Also removing empty line at the end of build-docs.

### CODE REVIEW

File: scripts/build-docs [hunk 1]
```diff
@@ -28,4 +28,3 @@ sphinx-apidoc -f -o docs/source oauth2client
 cd docs
 make html
 cd ..
-
```
Comment (nathanielmanistaatgoogle) [inline scripts/build-docs:31]: Is this change pertinent to this commit? If so, how? If not, drop it.

**Comment by nathanielmanistaatgoogle** [comment]
How sure are we that that's the file we wish to use? Are we following another project's example?

File: scripts/build-docs (hunk 1 above)
Comment (dhermes) [inline scripts/build-docs:31]: Nope. Will do.

**Comment by dhermes** [comment]
I was just basing it off of the contents of `docs/_static/google_logo.png`

**Comment by dhermes** [comment]
I removed the superfluous change and sent it as #236

**Pull request merged**